mobile_verifier/src/settings.rs

```diff
@@ -20,6 +20,7 @@ pub struct Settings {
     pub ingest: file_store::Settings,
     pub data_transfer_ingest: file_store::Settings,
     pub output: file_store::Settings,
+    pub data_sets: file_store::Settings,
```

Comment:
Can we get some comments on these settings?
`data_sets` makes some sense in the boostings modules. But just looking at settings, its hard to suss out what type values should be going here.

Same with `data_sets_directory`. 

It seems like these two settings should be closer together. They're more closely related to each other than other settings of the same type.

Reply:
When you say closer together, do you mean physically fewer lines between them or bundled together in one sub-section?

Reply:
I think both would achieve the same goal. :+1: